src/provider/cloud-element-templates/ElementTemplates.js

```diff
@@ -9,9 +9,9 @@ import { default as DefaultElementTemplates } from '../element-templates/Element
  * Registry for element templates.
  */
 export default class ElementTemplates extends DefaultElementTemplates {
-  constructor(templateElementFactory) {
-    super();
-    this._templates = {};
+  constructor(templateElementFactory, commandStack) {
+    super(commandStack);
+
```

Comment:
Let's assign the private property explicitly to not depend on the parent to set this:

```suggestion
    super(commandStack);

    this._commandStack = commandStack;
```